backend/benefit/applications/services/pdf_templates/base.html

```diff
@@ -4,21 +4,41 @@
 <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
 <head>
     <title>Ahjo PDF</title>
-    <style>
+   <style>
     table, td, th {
-        border: 1px solid black;
+        color: #1a1a1a;
+        padding: 7px;
+        line-height: 1.5;
+        font-size: 12px;
+    }
+    table th, .footer {
+        background-color: #0072c6;
+        color: white;
+    }
+    table tr:nth-child(odd){
+        background-color: #e8f3fc;
     }
     table {
         width: 100%;
         border-collapse: collapse;
-        text-align: center;
+        text-align: left;
     }
+
+    td.text-right, th.text-right {
+        text-align: right;
+    }
+
     body {
-        font-family: Arial, sans-serif;
+        font-family: ArialMT, sans-serif;
     }
     </style>
 </head>
 <body>
+<div>
+    <img src="https://www.hel.fi/wps/wcm/connect/b3717cdb-45a7-4575-946b-5ea799506c80/1/HELSINKI_Tunnus_MUSTA.png?MOD=AJPERES" width="80" height="37">
```

Comment:
The POs have told that a hel.fi re-implementation is going on, this URL probably would change. Though I don't know where this file should be stored - does the pdf renderer find it if it's stored on local disk?

Reply:
Unfortunately no, the pdfkit won't be able to render the file if src is relative path to the template. The only thing that works is backend hosting the image as a static file, then we use the url to that file here, but it should be over complicated as URL are different between env. 

I think we can fix it later when we launch the production, then we can host this logo in cloud storage and replace this with the URL there.

I'll leave this as it for now